CONTRIBUTING.md

```diff
@@ -1,139 +1,138 @@
 
-Alternatively, you can add an `.only` to a `describe` or `it` block (i.e. `describe.only('my spec')`)
-and run the `npm run test` command. Keep in mind that this will also run linters and aXe accessibility tests.
+If your idea has already been suggested and fulfills your business need, upvote that feature request with a thumbs up emoji (👍) and comment on the issue to let us know why you need this feature request and any other supporting information. We review the number of upvotes (represented by 👍) to help us prioritize feature requests. 
 
-To run all of the unit tests, run `npm run test:unit`.
+If you want to find other feature requests open for upvoting, check out our [feature request view sorted by status](https://github.com/orgs/uswds/projects/8/views/18?sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Status). 
 
-### Building
+#### 2. Submit an issue
 
-To build the `uswds` package in preparation for releases, run:
+If your idea is not in the [current issues backlog](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22), submit an issue using the [feature request template](https://github.com/uswds/uswds/issues/new?assignees=&labels=type%3A+feature+request&template=feature_request.md&title=).  A USWDS core team member may reach out to you if we need further clarification on your submission. 
 
-```sh
-npm run release
-```
 
-## Coding guidelines
+### 🛠 Submitting pull requests
 
-The purpose of our coding styleguides are to create consistent coding practices across 18F. The styleguide should be treated as a guide — rules can be modified according to project needs.
+If you want to propose a code fix, even better! Here’s a few steps to help you submit pull requests.
 
-This project follows the 18F Front End Guide [CSS](https://pages.18f.gov/frontend/#css) and [JavaScript](https://pages.18f.gov/frontend/#javascript). Please use this guide for your reference.
+#### If it’s a bug fix:
 
-## Browser support
-See [browser support](https://designsystem.digital.gov/documentation/developers/#browser-support) in the “Developer Documentation”.
+1. Check our [open issues backlog](https://github.com/uswds/uswds/issues) for any duplicate or similar issues. 
+2. If your bug has already been submitted, feel free to comment and provide additional context (if different than the original submission).  
+3. If your proposed fix is not in the open issues backlog, create an [issue](https://github.com/uswds/uswds/issues/new/choose) for the change you’re proposing.  This helps us track our work. 
+4. Fork this repo into your GitHub account (or just clone it if you’re an 18F team member). Read more about forking a repo on [GitHub](https://help.github.com/articles/fork-a-repo/). 
+5. Create a branch from `develop` that lightly defines what you’re working on (for example, add-styles).
+6. Ensure that your contribution works via npm, if applicable.
```

Comment:
>6. Ensure that your contribution works via npm, if applicable.

Not sure what this means. Does no. 6 refer to the code affecting `npm install @uswds/uswds` or?

Reply:
I wonder if it is trying to say something like "Ensure that your contribution runs locally (via npm)"? Not sure why it would be "if applicable"? 

@mejiaj is it safe to delete this line for now? Maybe in the future we can use this spot to point users to coding standards/guidelines/expectations?

Reply:
I think we should remove that bullet point entirely

Reply:
Done